Dockerfile.rhel7

```diff
@@ -1,10 +1,10 @@
-FROM registry.svc.ci.openshift.org/ocp/builder:golang-1.10 AS builder
-WORKDIR /go/src/github.com/openshift/cluster-ingress-operator
+FROM registry.svc.ci.openshift.org/ocp/builder:golang-1.12 AS builder
+WORKDIR /go
 COPY . .
-RUN make build
+RUN make GOPATH="" build
 
-FROM registry.svc.ci.openshift.org/ocp/4.0:base
-COPY --from=builder /go/src/github.com/openshift/cluster-ingress-operator/ingress-operator /usr/bin/
+FROM registry.svc.ci.openshift.org/openshift/origin-v4.0:base
```

Comment:
Why does `Dockerfile.rhel7` switch to the Origin image?

Reply:
Copy/paste error, fixed